plugins/airbrake/src/api/ProductionApi.ts

```diff
@@ -15,13 +15,17 @@
  */
 
 import { Groups } from './airbrakeGroups';
-import { AirbrakeApi } from './AirbrakeApi';
+import { AirbrakeApi, NoProjectIdError } from './AirbrakeApi';
 import { DiscoveryApi } from '@backstage/core-plugin-api';
 
 export class ProductionAirbrakeApi implements AirbrakeApi {
   constructor(private readonly discoveryApi: DiscoveryApi) {}
 
   async fetchGroups(projectId: string): Promise<Groups> {
+    if (!projectId) {
```

Comment:
see comment about doing this on the calling side instead, i really don't think this api should handle a completely faulty input like this